packages/typography/index.demo.tsx

```diff
@@ -0,0 +1,36 @@
+import { h, Component } from 'skatejs';
```

Comment:
I guess we can remove demo files as well and put all demos directly here to shrink amount of files in this module... what do you think?